.github/workflows/run-simple-integration-test.yml

```diff
@@ -0,0 +1,28 @@
+# Summary:
+# Test the GitHub Action using an integration test.
+# Makes sure the GitHub Action works properly when running on a clean machine, without building anything (integration test).
+# This integration test will break if the Vercel project or deployment are deleted, or if the VERCEl_TOKEN doesn't have access to the deployment url.
+#
+# See https://github.com/actions/checkout https://github.com/actions/checkout/releases/tag/v2
+
+name: 'GitHub Action integration test'
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+      - 'releases/*'
+
+jobs:
+  run-integration-test:
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: ./
+        with:
+          command: "echo \"Deployed url: https://my-app.vercel.app\""
```

Comment:
This isn't really an integration test, shouldn't it rather really deploy something to vercel?

Even if it's a simple HTML page.

Reply:
Okay it has been done